main/sketch.cpp

```diff
@@ -277,6 +277,7 @@ void loop() {
     // https://stackoverflow.com/questions/66278271/task-watchdog-got-triggered-the-tasks-did-not-reset-the-watchdog-in-time
 
     //     vTaskDelay(1);
+    digitalWrite(PIN_LED_FRONT, HIGH);
```

Comment:
this should not be committed on this branch, rather you made this change on the body-LED branch in PR #25. Please remove it from this branch, to do this run this command in GitBash:
```bash
git checkout main -- main/sketch.cpp
```